docs/learning/tutorials/showcase-to-viewer.md

```diff
@@ -0,0 +1,141 @@
+# Adding showcase widgets to your iTwin Viewer
+
+### Prerequisites
+
+This tutorial assumes that you already have:
+
+- Explored the [sample showcase](https://www.itwinjs.org/sample-showcase).
+- Your own local source for the iTwin Web Viewer based on the template @bentley/itwin-viewer
```

Comment:
Could add a link to itwin-viewer repo

Reply:
Added link to itwin-viewer repo on 'local source'